cucumber.cjs

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    default: {
+        require: ["tests/e2e/**/*.ts"],
+        requireModule: ["ts-node/register"],
+        retry: process.env.RETRY || 0,
+        format: ["@cucumber/pretty-formatter"]
+    }
+}
+
+// see https://github.com/cucumber/cucumber-js/blob/main/docs/configuration.md for documentation
```

Comment:
@saw-jan since this is your project, do you have any preference? The way it is currently done is in the same fashion as it's done for [web-app-presentation-viewer](https://github.com/JankariTech/web-app-presentation-viewer)

Reply:
@saw-jan I opened a separate issue for this. If you have any feedback or suggestions on your preference how this should be done in your project, please write them into [this issue](https://github.com/saw-jan/web-app-3dmodel-viewer/issues/18)